igarss_2023/igarss_mtedgecumbe_ts_analysis.ipynb

```diff
@@ -46,38 +46,40 @@
    "source": [
     "## 0. Initial Setup\n",
     "\n",
-    "To run this notebook, you'll need a conda environment with the required dependencies. You can set up a new environment (recommended) and run the jupyter server like:\n",
-    "```shell\n",
-    "conda create -n hyp3-mintpy python=3.8 asf_search hyp3_sdk \"mintpy>=1.3.2\" pandas jupyter ipympl\n",
+    "To run this notebook, you'll need to be in the `insar_analysis` conda environment within OpenSARLab.\n",
     "\n",
-    "conda activate hyp3-mintpy\n",
-    "jupyter notebook igarss_mtedgecumbe_ts_analysis.ipynb\n",
+    "Alternatively, you can set upon your own environment by running these commands in your shell:\n",
+    "```shell\n",
+    "curl -OL https://raw.githubusercontent.com/ASFOpenSARlab/opensarlab-envs/main/Environment_Configs/insar_analysis_env.yml\n",
+    "conda env create -f insar_analysis_env.yml\n",
     "```\n",
-    "Or, install these dependencies into your own environment:\n",
+    "Then launch this notebook from the new environment:\n",
     "```shell\n",
-    "conda install hyp3-mintpy python=3.8 asf_search hyp3_sdk \"mintpy>=1.3.2\" pandas jupyter ipympl\n",
-    "jupyter notebook igarss_mtedgecumbe_ts_analysis.ipynb\n",
+    "conda activate insar_analysis\n",
+    "jupyter lab igarss_mtedgecumbe_ts_analysis.ipynb\n",
     "```\n",
     "\n",
     "Then we'll start the data processing"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "821027c4-f589-4a5f-a099-55790e5a6bce",
+   "id": "9e5c5b0b",
    "metadata": {},
    "source": [
-    "First import the needed modules and set up our processing directories"
+    "## 1. Request On Demand InSAR products from ASF HyP3\n",
+    "\n",
+    "A major step towards working with SAR data at scale is learning how to download and request the creation of data programmatically. Accomplishing these tasks via code makes it much easier to request large amounts of data and to make similar requests in the future.\n",
+    "\n",
+    "To request the generation of an Interferometric Synthetic Aperture Radar (InSAR) product from the Alaska Satellite Facility (ASF), you can follow the general steps outline in the cells below. For this example, we'll be requesting the generation of an interferogram that shows inflation-related displacement at Mt. Edgecumbe:"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "9e5c5b0b",
+   "id": "ddd24a07-2665-4bdc-83e0-bedc4f33c7fe",
    "metadata": {},
    "source": [
-    "## 1. Request On Demand InSAR products from ASF HyP3\n",
-    "\n",
-    "Use your [NASA Earthdata login](https://urs.earthdata.nasa.gov/) to connect to [ASF HyP3](https://hyp3-docs.asf.alaska.edu/)."
+    "1. Create an account: If you haven't already, create an [NASA Earthdata login](https://urs.earthdata.nasa.gov/). This will allow you connect to ASF HyP3 via the Python SDK. Running the cell below will prompt you for Earthdata username and password"
```

Comment:
```suggestion
    "1. **Create an account**: If you haven't already, create an [NASA Earthdata login](https://urs.earthdata.nasa.gov/). This will allow you connect to ASF HyP3 via the Python SDK. Running the cell below will prompt you for your Earthdata username and password"
```